benchbuild/source/base.py

```diff
@@ -0,0 +1,206 @@
+"""
+Provide a base interface for downloadable sources.
+"""
+import abc
+import itertools
+import typing as tp
+
+import attr
+import plumbum as pb
```

Comment:
Codacy found an issue: [Unable to import 'plumbum'](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5694346)